### PR TITLE
Update new AWS docs

### PR DESCRIPTION
**Pull request by jdmooney**
Updating how-tos/add-a-certificate and tutorials/single-tier based on latest comments

### CODE REVIEW

**Review by sbrossie — APPROVED** [review]

File: userguide/aws/how-tos/add-a-certificate.adoc [hunk 1]
```diff
@@ -54,17 +56,17 @@ In either case you will be taken to the main page of the ACM. Initially this wil
 
 The next page will give you the option to request a public or private certificate. The private option may be grayed out. Click *Request a Public Certificate*. The page that appears will ask you to specify several parameters, including one or more domain names and a validation method.
 
-image::https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ELB-request-certificate.png[align=center]
+image::../../assets/aws/ELB-request-certificate.png[align=center]
```
Comment (ghost) [inline userguide/aws/how-tos/add-a-certificate.adoc:59]: @jdmooney - you only need one set of "../" to indicate the relative path: 

`image::../assets/aws/ELB-request-certificate.png[align-center]`

File: userguide/aws/tutorials/single-tier.adoc [hunk 2]
```diff
@@ -55,7 +55,7 @@ When the key pair is generated, click *Launch Instances*. You should see the scr
 
 image::../../assets/aws/single-tier-launching.png[align=center]
 
-Your instance is finally launching! To follow what is happening on the EC2 Dashboard, scroll all the way down to the bottom, and click *View Instance* at the bottom right. This will take you to the *Instances* screen which is part of the EC2 Dashboard.
+Your instance is finally launching! To follow what is happening on the EC2 Dashboard, scroll all the way down to the bottom, and click *View all instances* at the bottom right. This will take you to the *Instances* screen which is part of the EC2 Dashboard.
 
 image::../../assets/aws/single-ami-instances.png[align=center]
```
Comment (FlowWriter) [inline userguide/aws/tutorials/single-tier.adoc:60]: Same comment about relative path (see add-a-certificate.adoc).

File: userguide/aws/tutorials/single-tier.adoc [hunk 3]
```diff
@@ -31,7 +31,7 @@ Click *Continue to Subscribe*. The next page will give the AWS Terms and Conditi
 
 The next page will give several configuration options.
 
-image::../../assets/aws/single-ami-configure.png[align=center]
+image::../../assets/aws/single-ami-configuration.png[align=center]
```
Comment (ghost) [inline userguide/aws/tutorials/single-tier.adoc:34]: Same comment about relative path (see add-a-certificate.adoc).

**Review by ghost — CHANGES_REQUESTED** [review]
Since we haven't yet done our first pass for set-up-a-multi-tier-system.adoc, I recommend starting a new PR for it. @jdmooney - do you know how to uncommit a file out of a PR and send it to a different (new) branch? LMK if you need help with that.

File: userguide/aws/tutorials/single-tier.adoc (hunk 3 above)
Comment (jdmooney) [reply userguide/aws/tutorials/single-tier.adoc:34]: See above

File: userguide/aws/how-tos/add-a-certificate.adoc (hunk 1 above)
Comment (jdmooney) [reply userguide/aws/how-tos/add-a-certificate.adoc:59]: The file is in userguide/aws/howtos and the images are in userguide/assets. So I  think the link is correct. And it works on my branch.

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
@reshmabidikar - Can you please merge this PR?

**Comment by reshmabidikar** [comment]
> @reshmabidikar - Can you please merge this PR?

Done!

**Pull request merged**